### PR TITLE
Remove schema from x-scheme-handler/file files

### PR DESCRIPTION
**Pull request by leiserfg**
Solves 
https://github.com/chmln/handlr/issues/21

### CODE REVIEW

File: src/common/mime_types.rs [hunk 1]
```diff
@@ -29,6 +29,10 @@ impl TryFrom<&str> for MimeType {
 
     fn try_from(arg: &str) -> Result<Self> {
         if let Ok(url) = url::Url::parse(arg) {
+            if url.scheme() == "file" {
+                print!("{}", url.path());
```
Comment (chmln) [inline src/common/mime_types.rs]: The print here seems like a debugging leftover ?

**Review by chmln — COMMENTED** [review]
see comment above

File: src/common/mime_types.rs (hunk 1 above)
Comment (leiserfg) [reply src/common/mime_types.rs]: Yes, it was, sorry.

Comment (leiserfg) [reply src/common/mime_types.rs]: Any update? without this, I can't use `handlr` with telegram-desktop.

**Pull request merged**